drivers/internal/TableCRC.cpp

```diff
@@ -16,11 +16,9 @@
  */
 
 #include <stdint.h>
-#include "drivers/TableCRC.h"
+#include "drivers/internal/TableCRC.h"
```

Comment:
Yes, `TableCRC.h` is in the correct place, but logically, `TableCRC.cpp` should be inside `source`, as it's not required to be visible on the path. It would be part of the stuff that would be replaced by a binary.

It could be either `source/TableCRC.cpp` or `source/internal/TableCRC.cpp`. The latter would seem kind of logical, but I don't have a strong view, as long as it's in `source`.

Reply:
I don't have a strong view on this either but I'd prefer all the source files under `source`, so let's keep `source/TableCRC.cpp`